lib/util/ios-proxy.js

```diff
@@ -91,73 +101,82 @@ let iOSProxy = {
     }
   }),
 
-  checkIfRunning() {
+  checkIfRunning: task.async(function*() {
     // Check for any currently running instances of ios_webkit_debug_proxy.
     let running = false;
-    let cmd = "/usr/bin/pgrep";
-    let name = "ios_webkit_debug_proxy";
-    let args = ["-x"];
+    let cmd = "/usr/bin/pgrep -x ";
```

Comment:
Hmm, so `runInShell` takes an `args` array, but you've folded the `-x` into the command? Is the args array not really needed then?